lib/console.js

```diff
@@ -166,6 +170,36 @@ Console.prototype.assert = function assert(expression, ...args) {
   }
 };
 
+// Defined by: https://console.spec.whatwg.org/#clear
+Console.prototype.clear = function clear() {
+  // It only makes sense to clear if _stdout is a TTY.
+  // Otherwise, do nothing.
+  if (this._stdout.isTTY) {
+    const { cursorTo, clearScreenDown } = require('readline');
+    cursorTo(this._stdout, 0, 0);
+    clearScreenDown(this._stdout);
+  }
+};
+
+// Defined by: https://console.spec.whatwg.org/#count
+Console.prototype.count = function count(label = kDefaultCount) {
+  // Curiously, the spec is silent on whether and how these counters
+  // get reset. Within the browser, that likely is not so much of a
+  // problem but it could be a source of nasty memory leaks in a
+  // Node.js process. We could add an option for clearing the counters
+  // but it would be non-standard.
```

Comment:
That is indeed a problem. Can we file an issue at WHATWG, or supply a `console.countReset(label)` as an extension?